anesthetic/read/cobaya.py

```diff
@@ -21,8 +21,14 @@ def read_paramnames(root):
         paramnames = header.split()[2:]
         try:
             from getdist import loadMCSamples
-            s = loadMCSamples(file_root=root)
+            s = loadMCSamples(file_root=root, no_cache=True)
```

Comment:
Needed to add this, otherwise the GetDist cache will eat up my remote disk space.